core/templates/dev/head/components/ExplorationSummaryTileDirective.js

```diff
@@ -61,6 +70,8 @@ oppia.directive('explorationSummaryTile', [function() {
           }
           return result;
         };
+
+        $scope.showBriefContributors = false;
```

Comment:
Not sure what this means, from reading the name. $scope.allContributorsAreVisible, perhaps?